src/main/java/uk/gov/companieshouse/overseasentitiesapi/controller/OverseasEntitiesController.java

```diff
@@ -0,0 +1,14 @@
+package uk.gov.companieshouse.overseasentitiesapi.controller;
```

Comment:
Probably we can use an `HealthcheckController` instead

Reply:
the inclusion of the spring-boot-starter-actuator in the pom will automatically create a /actuator/health endpoint for the app that will return the health status.
I figured we would at least need a minimum of a post endpoint, or at least something to prove our controller was working.